recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "1.4.0" %}
-{% set tag = "-1" %}  # exceptionally for the 1.4.0 release to match the pypi release
+{% set version = "1.4.1.post1" %}
```

Comment:
Why do we need `.post1` here?

Reply:
Long story short I fucked up 1.4.1 🙂. Long story is that we thought it would better to unpin numpy upper bound but realize that would not be the best (basically we don't have a perfect situation) but I already tagged. So we put back a upper bound and make a post tag and thus this release version number. It looks the least painful process.

Reply:
OK, I let someone else merge when this LGTT.

Reply:
We messed up (rather than just you) since I was there as well :wink: and did not remember Thomas' comment early enough but oh well this is not a huge deal at all.